app/views/home/dashboard.scala.html

```diff
@@ -31,7 +31,7 @@
 
 @confirmAllocationExpired = @{
     allocationDetails match {
-        case Some(AllocationDetails(_, _, _, Some(expirationDate))) if LocalDate.now().isAfter(expirationDate) => true
+        case Some(AllocationDetails(_, _, _, Some(expirationDate))) if LocalDate.now().isAfter(expirationDate) && usr.application.get.applicationStatus == ALLOCATION_EXPIRED => true
```

Comment:
Shall we get rid of the date check entirely? Feels like now the back end is making the decision the frontend doesn't need to be checking/interfering with its work